js/libs/form.js

```diff
@@ -133,6 +133,16 @@ Fliplet.Widget.instance('form-builder', function(data) {
 
           var showCurrentDateTime = field.autofill === 'always';
 
+          if (field._type === 'flCheckbox') {
+            if (!Array.isArray(fieldData)) {
+              fieldData = _.compact([fieldData]);
+            }
+          } else {
+            if (Array.isArray(fieldData)) {
```

Comment:
Wouldn't this break checkboxes and files?

Reply:
It shouldn't break checkboxes because checkbox fields won't come to this branch of the code, but files (and images) might need to be excluded too.